### PR TITLE
Solution for #794

### PR DESCRIPTION
**Pull request by Portagoras**
Basic implementation for the accuracy indicator for the Photo Map. I am not to sure if I implemented at the right position, please take a look.
I also added comments so the functions make sense and are visible in the various IDE Inspectors.

### CODE REVIEW

**Comment by Portagoras** [comment]
I see what I can do later today. Thanks for the hint. I don't know all the code yet, so this might happen more often.

**Comment by kylecorry31** [comment]
Thanks!

**Comment by Portagoras** [comment]
I just noticed, that the location of the class file might need to be adjusted to the other layers. It runs fine for me but best quickly test it by running around a bit..

**Comment by kylecorry31** [comment]
I believe the best spot for the new layer will be here: https://github.com/kylecorry31/Trail-Sense/tree/main/app/src/main/java/com/kylecorry/trail_sense/navigation/ui/layers

File: app/src/main/java/com/kylecorry/trail_sense/tools/maps/ui/PhotoMapView.kt [hunk 1]
```diff
@@ -3,8 +3,10 @@ package com.kylecorry.trail_sense.tools.maps.ui
 import android.content.Context
 import android.graphics.*
 import android.util.AttributeSet
+import android.util.Log
```
[the remark this answers is not in the document]
Comment (Portagoras) [reply app/src/main/java/com/kylecorry/trail_sense/tools/maps/ui/PhotoMapView.kt:6]: Yes I can do so!

[the remark this answers is not in the document]
Comment (kylecorry31) [reply app/src/main/java/com/kylecorry/trail_sense/tools/maps/ui/PhotoMapView.kt:6]: Thanks!

**Comment by kylecorry31** [comment]
Thanks, this looks good!

**Pull request merged**